airbyte-integrations/bases/s3-destination-base-integration-test/src/main/java/io/airbyte/integrations/destination/s3/S3BaseJsonlDestinationAcceptanceTest.java

```diff
@@ -52,6 +54,11 @@ protected List<JsonNode> retrieveRecords(final TestDestinationEnv testEnv,
     return jsonRecords;
   }
 
+  @Override
+  public void testEntrypointEnvVar() throws Exception {
```

Comment:
this seems like an unnecessary override given that it's just calling the same method from the parent class, which it is overriding